AppleMusic/Source/View/RadioView/FavoritesStationView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  FavoritesStationView.swift
+//  AppleMusic
+//
+//  Created by Artem Muho on 20.02.2022.
+//
+
+import SwiftUI
+
+struct FavoritesStationView: View {
+    @State var rowsData = FavoritesStationModel.getRows()
+    
+    @State var rows = [
+        GridItem(.flexible())
+    ]
+    var body: some View {
+        LazyHGrid(rows: rows) {
+            ForEach(rowsData, id: \.self) { row in
+                VStack(alignment: .leading) {
+                    Text(row.type)
+                        .foregroundColor(.gray)
+                        .font(.subheadline)
+                        .bold()
+                        .textCase(.uppercase)
+                    Text(row.title)
+                        .font(.title)
+                    Text(row.disription)
+                        .font(.title)
+                        .foregroundColor(.gray)
+                    row.image
+                        .resizable()
+                        .frame(width: 360, height: 240)
```

Comment:
Я бы здесь не задавал жестко размеры. Лучше исходить из ширины экрана. Например, вот так:
```suggestion
                        .scaledToFit()
                        .frame(width: UIScreen.main.bounds.width - 40)
```